src/Components/People.js

```diff
@@ -0,0 +1,64 @@
+import { Component } from "react";
+import axios from "axios";
+
+class People extends Component {
+  state = { input: "", person: {}, searched: false };
+
+  handleSubmit = async (e) => {
+    e.preventDefault();
+    this.setState({ searched: true });
+    try {
+      const { input } = this.state;
+      const res = await axios.get(
+        `https://ghibliapi.herokuapp.com/people/${input}`
```

Comment:
You have to read the api documentation carefully. It searches people by ID, not by name. You'll need to get all of the people and then search through them for if a name matches your input